packages/shopify-codemod/test/fixtures/global-assignment-to-default-export/local-non-prototype-assignment.input.js

```diff
@@ -0,0 +1,5 @@
+App.A = class A {
+
+};
+
+App.A.foo = () => {};
```

Comment:
Good call.  That clears up a few more transform errors.  Updated, let me know what you think.